config.yaml

```diff
@@ -2,6 +2,7 @@ destination: &dest syn12177492
 staging_path: ./staging
 gx_folder: syn52948668
 gx_table: syn60527066
+team_images_id: syn12861877
```

Comment:
We won't have team images for Model-AD. 

What happens if we omit the `team_images_id`  config entirely, or leave its value blank? Will ADM generate the file without that key, or with a blank/null value? 

Is it worth adding a test case to cover that scenario?

Reply:
Ah, good point. Will you still want the `dataversion` file, just without the `team_images_id`?

Reply:
I pushed changes that will do this - let me know if that works. A test `dataversion.json` file was uploaded [here](https://www.synapse.org/Synapse:syn64713011.3) without the `team_images_id `.

Reply:
@BWMac That should enable us to update ADM to conditionally load image files.  @hallieswan fyi